subprojects/testing-jvm-infrastructure/src/test/groovy/org/gradle/api/internal/tasks/testing/junit/JUnitTestClassProcessorTest.groovy

```diff
@@ -34,11 +35,11 @@ class JUnitTestClassProcessorTest extends Specification {
     @Rule TestNameTestDirectoryProvider tmp = new TestNameTestDirectoryProvider(getClass())
 
     def processor = Mock(TestResultProcessor)
-    def spec = new JUnitSpec([] as Set, [] as Set, [] as Set, [] as Set, [] as Set)
 
-    @Subject classProcessor = withSpec(spec)
+    @Subject classProcessor = createProcessor([] as Set, [] as Set, [] as Set, [] as Set, [] as Set)
```

Comment:
Even though it's perfectly valid, every time I see annotations on a field that doesn't have `def` or an access modifier, or a type, my first reaction is "This must be wrong!"  I'm surprised we don't express a strong opinion via CodeNarc on this.